### DISCUSSION TITLE
How to trigger custom plugin only for specific routes

### DISCUSSION THREAD

**Discussion by djukas95** [Kong/kong#10166] [Help] [answered]
Guys, i have issue with Kong custom plugin.
I need to enable this custom plugin for **2 specific routes** but for other routes i need to use Kong as usual.

For example:

**example.com/app/v1/bot
example.com/app/v1/bot/fresh/agt/reply**

for this 2 routes i need to enable my plugin, but for other routes for example (example.com/app/v1/xyz) i need to use Kong as usual.

**Comment by locao** [comment, marked as answer]
Please take a look at this doc: [create plugin associated to a specific route](https://docs.konghq.com/gateway/latest/admin-api/#create-plugin-associated-to-a-specific-route).
